### ISSUE TITLE
How to get the supabase functions for user controls to work

### ISSUE THREAD

**Issue by eithe** [marmelab/atomic-crm#68]
Hi, I'm trying to get the supabase/functions/updatePassword and users/index to work, but all I get is a 503 from the endpoints, even when the browser is sending the OPTIONS call. Trying to figure out what I'm missing, but haven't been able to thus far. The postmark functions like addNoteToX works.

I'm running locally (make start). Any tips?

**Comment by jonathan-marmelab** [comment]
Hello,

Do you have any additional information about the error from the supabase edge runtime container ?

Can you try the run the following command, to force the deploy of the edge functions:

```bash
make start-supabase-functions
```

**Comment by eithe** [comment]
Thanks, running `make start-supabase-functions` makes it work. Should I just have that running in a separate window or is it supposed to start with `make start`? If the latter, do you have any tips on what I could do to troubleshoot it not starting?

**Comment by jonathan-marmelab** [comment]
On my computer, the edge functions service starts automatically, so I did not face your problem. 

The `make start-supabase-functions` is blocking so you must start it in a separate window (this is why we did not add it to the the `make start`).

**Comment by fzaninotto** [comment]
The `make start` command should start the entire Supabase stack locally, including Edge functions. It works properly on numerous computers, either Linux of MacOS, in our tests. If it doesn't work on your side, this may hide something else. 

1. Can you tell us more about your environment (OS, Supabase CLI version)?
2. Did you update the `supabase/config.toml` config file? 
3. Can you post a minimal diff of your code with the `main` branch so we can try and reproduce your issue?

**Comment by eithe** [comment]
Appreciate the quick and thorough feedback!

I'm embarrassed to say that I restarted my MBP, and now the edge functions starts with `make start`. Sorry guys ... I didn't even do what we normally tell our parents to do when they have technical issues :/

When it didn't work yesterday I quickly and incorrectly assumed I had to do something special to get the whole email + user editing stuff up and running, but I guess something just had hung from previous runs.

Really appreciate this extensive example, truly great work!
